server/controllers/itemController.js

```diff
@@ -45,22 +45,23 @@ module.exports = {
     });
   },
   sellItem(req, res, next) {
-    console.log(req);
-    console.log(req.user);
     const newItem = req.body;
+
+    const sellerId = (req.user ? req.user.user.id : 39);
+
     newItem.images = JSON.stringify(newItem.images);
     console.log('Creating new item,', newItem);
     db.items.create(newItem)
     .then(product => {
       db.items.getById(product[0])
       .then((result) => {
+        const id = result[0].id;
+        const transaction = { item_id: id, buyer_id: null, seller_id: sellerId };
```

Comment:
I think you need to access the result's id through ['id']? if this works whatever